app/models/location.py

```diff
@@ -0,0 +1,27 @@
+from .. import db
+
+
+class ZIPCode(db.Model):
+    __tablename__ = 'zip_codes'
+    id = db.Column(db.Integer, primary_key=True)
+    zip_code = db.Column(db.String(5), unique=True)
+    users = db.relationship('User', backref='zip_code', lazy='dynamic')
+    addresses = db.relationship('Address', backref='zip_code', lazy='dynamic')
+
+    def __repr__(self):
+        return '<ZIPCode \'%s\'>' % self.zip_code
+
+
+class Address(db.Model):
+    __tablename__ = 'addresses'
+    id = db.Column(db.Integer, primary_key=True)
+    # USPS Addressing Standards: http://pe.usps.com/text/pub28/28c2_001.htm
+    recipient_line = db.Column(db.Text)         # ABC MOVERS
+    delivery_address_line = db.Column(db.Text)  # 1500 E MAIN AVE STE 201
```

Comment:
Should we just change "deliver_address_line" to "street_address"? feels more readable and this address is not really a "delivery address line" because it's not a place that has stuff delivered to it